graphs/possible_bipartition.py

```diff
@@ -5,8 +5,33 @@ def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(N+E) N-number of nodes, E -number of edges
+        Space Complexity: O(N)??
     """
-    pass
+    if not dislikes:
+        return True
 
+    queue = deque()
+    visited = set()
+    group1 = set()
+    group2 = set()
+
+    queue.append(0)
```

Comment:
Your two test cases are failing because you need a guard clause to find the first node in the list that is connected to other nodes in the graph. 

Say you have a graph such as this:
        `dislikes =  [ [],
          [2, 3],
          [1, 3],
          [1, 2]
        ]`
If you draw out the graph, you will see that Node 0 is disconnected from the rest of the graph - it has no edges. Because it has no edges/neighbors, nothing else will be added to your queue after you pop Node 0 off the queue and you will return `True` by default.